ape_aws/accounts.py

```diff
@@ -38,6 +38,10 @@ class KmsAccount(AccountAPI):
     key_id: str
     key_arn: str
 
+    @property
+    def alias(self) -> str:
+        return self.key_alias
```

Comment:
Wonder if it might make sense to remove `alias/` from here 
```suggestion
        return self.key_alias.strip("alias/")
```